docs/developer/PROW_CI.md

```diff
@@ -64,12 +64,278 @@ The jobs run can be seen in PRs and in the links in the README.md file of OADP r
 
 OADP operator master branch is tested against the last 3 minor OCP releases. To Update an OCP version the project is tested against, see [Update OCP version](#update-ocp-version).
 
-### TODO Creating new release branch
+### Creating new release branches
 
-https://github.com/openshift/oadp-operator/issues/1227
+To create new OADP release branches (they must follow the pattern `oadp-major.minor`, [example](https://github.com/openshift/oadp-operator/tree/oadp-1.4)):
+- First, create CI files for each new repository branch.
+- Create the release branches in each one of related repositories of OADP, and OADP operator repository itself.
+- The new release branches must be updated:
+  - Downstream repositories release branches may not need updates (except OADP operator, which always need updates).
+  - Upstream repositories release branches need to be rebased.
+
+> **Note**: Try to always create release branches from default branches (usually called master).
```

Comment:
Could we have a little help here:

The default branch of a given repository can be checked using the following command
```sh
$ curl -s https://api.github.com/repos/openshift/oadp-operator | jq -r .default_branch
master
```

Reply:
this can be checked through UI, do not think we need this

Reply:
It can be checked in many way of course. But with new branch you usually use shell or UI? either one works if it's the one to continue. I believe we don't want the user to switch between UI and shell, rather have set of instructions within one.